### PR TITLE
RHEL minimal image

### PR DESCRIPTION
**Pull request by dacleyra**
@arthurdm 

Use the new minimal image

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/getting_started_with_containers/using_red_hat_base_container_images_standard_and_minimal#building_container_images_with_the_minimal_base_image

### CODE REVIEW

**Comment by arthurdm** [comment]
looks good to me, thanks @dacleyra!

CC @dinogun for review

**Comment by arthurdm** [comment]
@dacleyra / @dinogun -this image is currently beta - so let's wait on this PR a bit.

File: ibmjava/update.sh [hunk 1]
```diff
@@ -154,14 +154,8 @@ EOI
...
-RUN yum makecache fast \
-    && yum update -y \
-    && yum -y install openssl wget ca-certificates \
-    && yum clean packages \
-    && yum clean headers \
-    && yum clean all \
-    && rm -rf /var/cache/yum \
-    && rm -rf /var/tmp/yum-*
+RUN microdnf --enablerepo=rhel-7-server-rpms install openssl wget ca-certificates gzip tar && \
+    microdnf update; microdnf clean alls
```
Comment (qerub) [inline ibmjava/update.sh]: `error: Invalid clean argument: 'alls'`

Comment (samuel-hawker) [reply ibmjava/update.sh]: * `microdnf clean all`

Comment (dacleyra) [reply ibmjava/update.sh]: thanks, corrected

**Comment by arthurdm** [comment]
thanks @dacleyra 

I was wondering if we could put in 2 additional updates to this PR:
1. Make this as an additional Dockerfile to the existing RHEL one (instead of overriding the existing one)
2. Apply it against the IBM SFJ (small footprint java) instead of the full JRE - this would give us the smallest possible stack

**Comment by SamuelHawker** [comment]
Our team would like the full JRE, so could you make it for the JRE and the SFJ? :)

**Pull request closed without merging**